recipes/spiders/tudo_gostoso.py

```diff
@@ -41,6 +35,23 @@ def parse_vegan_and_veggie(self, response):
                 '.e-instructions p::text'
             ).getall()
 
+        verification_denied_words = []
+
+        for ingredient in ingredients:
+            verification_denied_words = ingredient.split()
+            for word in verification_denied_words:
+                if self.denied_words.count(word):
+                    return None
```

Comment:
Nao precisa escrever `return None`, o proprio fato de fazer `return` ja e `None`.

Reply:
Imaginei que somente o return estava fora da pep8, mas vou corrigir :D

Reply:
se for isso mesmo, me manda a doc pra eu ver, fiquei curioso.